docs/roles/ruby.md

```diff
@@ -10,10 +10,21 @@ Find a list of accepted version with `rbenv install -l`. Default is 2.4.0
 
 # Rails
 
+## Dependencies
+
 Simply add roles `nodejs` and `ruby` in your playbook.yml.
 Note that rails will not be installed unless specified in your Gemfile.
 
 Using mysql or postgres? then include `mysql` or `postgresql` role before `ruby`.
```

Comment:
I don't see the `ruby` role doing anything with MySQL or Postgresql, does that really make sense?